### PR TITLE
Json color ext points + extensions localization

### PR DESCRIPTION
**Pull request by CGNonofr**
and some bugfixes

### CODE REVIEW

File: src/rollup-vsix-plugin.ts [hunk 1]
```diff
@@ -96,19 +96,27 @@ export default function plugin (options: Options = defaultOptions): Plugin {
...
-      const usedFiles = ['package.json', ...extractPathsFromExtensionManifest(manifest.contributes)]
+      const usedFiles = ['package.json', 'package.nls.json', ...extractPathsFromExtensionManifest(manifest.contributes)]
+      const nlsExists = files['package.nls.json'] != null
 
-      const vsixFile: Record<string, Buffer> = usedFiles.reduce((acc, usedFile) => ({
-        ...acc,
-        [usedFile]: files[path.relative('/', path.resolve('/', usedFile))]!
-      }), {} as Record<string, Buffer>)
+      const vsixFile: Record<string, Buffer> = usedFiles.reduce((acc, usedFile) => {
+        const filePath = files[path.relative('/', path.resolve('/', usedFile))]
```
Comment (kaisalmen) [inline src/rollup-vsix-plugin.ts:103]: Does it make sense to use Platform specific path separators like `path.sep`?

**Review by kaisalmen — COMMENTED** [review]
Can I test this somehow?

File: src/rollup-vsix-plugin.ts (hunk 1 above)
Comment (CGNonofr) [reply src/rollup-vsix-plugin.ts:103]: I've published a 1.78.3-next.0

> Does it make sense to use Platform specific path separators like path.sep?

I don't think so... it's just some virtual internal paths

Comment (kaisalmen) [reply src/rollup-vsix-plugin.ts:103]: Ok.

**Review by kaisalmen — APPROVED** [review]
This looks good, but I can not verify it. We should think about how I can better verify more complex PRs.

**Comment by CGNonofr** [comment]
> I mean generally: How should I verify what there is works/makes sense

That's not simple... There is a lot of bigs changes but with time it will eventually stabilize

This PR is one of the easiest PR to review for a long time though (:

**Comment by kaisalmen** [comment]
> This PR is one of the easiest PR to review for a long time though (:

Didn't you say last week that there are no more bigger PRs coming up? 😆

**Comment by CGNonofr** [comment]
> > This PR is one of the easiest PR to review for a long time though (:
> 
> Didn't you say last week that there are no more bigger PRs coming up? 😆

I mean it's easier than the few previous ones :)

**Comment by kaisalmen** [comment]
> I mean it's easier than the few previous ones :)

Ok, got you 😅

**Comment by CGNonofr** [comment]
@kaisalmen added some new commits (starting from 929fd331bda1f89b82325625a23f306e142bf98e) and released a new [1.78.3-next.1](https://www.npmjs.com/package/@codingame/monaco-vscode-api/v/1.78.3-next.1) version

**Review by kaisalmen — APPROVED** [review]
I have tested the latest preview build `1.78.4-next.3` and there were no regressions. But, I have not tested both plugins and won't find time before next weekend. I don't want to block merging these things in.

Let's discuss/think about how we can test the existing features (unit tests or examples) in May. 🙂

**Comment by CGNonofr** [comment]
I won't be there a lot in may unfortunately , but I agree we need automated tests

**Comment by kaisalmen** [comment]
> I won't be there a lot in may unfortunately

Good to know

> but I agree we need automated tests

at least before next round of major refactorings 😉

**Pull request merged**